commander/test/utils/network_identifier.ts

```diff
@@ -18,12 +18,18 @@ import { getNetworkIdentifierWithInput } from '../../src/utils/network_identifie
 
 describe('network identifier utils', () => {
 	describe('getNetworkIdentifierWithInput', () => {
-		const mainnetNetworkIdentifier =
-			'5a59db36ca7de3cbb4bf27f95665e86952ccc66f8bf7ce8f6f6b3561b920f801';
-		const testnetNetworkIdentifier =
-			'10f236f6d00a8f565bbe43c4ef0e3818f488bf4abefbe041155a3d019ef9a947';
-		const defaultNetworkIdentifier =
-			'7777777777777777777777777777777777777777777777777777777777777777';
+		const mainnetNetworkIdentifier = Buffer.from(
+			'5a59db36ca7de3cbb4bf27f95665e86952ccc66f8bf7ce8f6f6b3561b920f801',
+			'hex',
+		).toString('base64');
+		const testnetNetworkIdentifier = Buffer.from(
+			'10f236f6d00a8f565bbe43c4ef0e3818f488bf4abefbe041155a3d019ef9a947',
+			'hex',
+		).toString('base64');
+		const defaultNetworkIdentifier = Buffer.from(
+			'7777777777777777777777777777777777777777777777777777777777777777',
+			'hex',
+		).toString('base64');
```

Comment:
Same here, if we are gonna use `base64` and we don't need to double convert. Just convert those once and use it everywhere as base64.

Reply:
Same as above